Tests/StreamChatTests/StateLayer/StateLayerDatabaseObserver_Tests.swift

```diff
@@ -203,6 +204,35 @@ final class StateLayerDatabaseObserver_Tests: XCTestCase {
         XCTAssertEqual(2, changeCount)
     }
     
+    // MARK: - Reusing Existing Items
+    
+    func test_reuseItems_whenSomeItemsChange_thenOthersAreReused() async throws {
```

Comment:
maybe some more tests would be nice, covering different models

Reply:
I'll add one for each of the model.